api/pin_reset.go

```diff
@@ -46,10 +48,9 @@ func (a *Api) SendPinReset(res http.ResponseWriter, req *http.Request, vars map[
 	var usrDetails *shoreline.UserData
 	var err error
 
-	var token = req.Header.Get(TP_SESSION_TOKEN)
-
-	td := a.sl.CheckToken(token)
-	if td == nil || td.IsServer {
+	if td := a.token(res, req); td == nil {
+		return
+	} else if td.IsServer {
```

Comment:
Does it mean we consider we will never need to make pin reset on behalf of a user ?